### PR TITLE
improve accessibility - tables

### PR DESCRIPTION
**Pull request by jiayinjx**
Improve accessibility for tables - Benefit Information/Payroll Information/Time and absence 
make the tables could be identified as a whole table and add brief description about what it is for.
TAB for the titles of headers.

### CODE REVIEW

File: hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp [hunk 1]
```diff
@@ -88,25 +88,25 @@
             <table class="dl-table table">
               <thead>
                 <tr rsf:id="header:">
-                  <th class="flc-pager-sort-header" rsf:id="name"><a href="javascript:;">Name</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="status"><a href="javascript:;">Status</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="start"><a href="javascript:;">Start</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="end"><a href="javascript:;">End</a></th>
-                  <th class="flc-pager-sort-header" rsf:id="total"><a href="javascript:;">Total</a></th>
+                  <th scope="col" class="flc-pager-sort-header" rsf:id="name"><a href="#">Name</a></th>
```
Comment (timlevett) [inline hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp]: Does having the `<a>` in there break the sorting mechanism that is in fluid sorting?

Comment (jiayinjx) [inline hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp]: not actually sure if <a> tag does.
it was original skip reading the table also all the content in the table.
research from:

http://webaim.org/techniques/tables/
http://www.w3.org/WAI/tutorials/tables/
http://www.w3.org/TR/html4/struct/tables.html#h-11.2.1

I want the table be recognized and let the screen reader reads out the information in the table in order.

Comment (timlevett) [inline hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp]: So in testing the application post your fix, did the sorting still work?

Comment (jiayinjx) [inline hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/timeAbsence.jsp]: I was only be able to test it in the inspector of browsers. it works.
could not test in local.

**Comment by vertein** [comment]
:+1:

**Pull request merged**